pyproj/_transformer.pyx

```diff
@@ -30,38 +45,41 @@ cdef class _Transformer:
         if self.projctx is not NULL:
             proj_context_destroy(self.projctx)
 
-    def set_radians_io(self):
-        self.input_radians = proj_angular_input(self.projpj, PJ_FWD)
-        self.output_radians = proj_angular_output(self.projpj, PJ_FWD)
+    def _set_radians_io(self):
+        self.input_radians = {
+            PJ_FWD: proj_angular_input(self.projpj, PJ_FWD),
+            PJ_INV: proj_angular_input(self.projpj, PJ_INV),
+            PJ_IDENT: proj_angular_input(self.projpj, PJ_IDENT),
+        }
+        self.output_radians = {
+            PJ_FWD: proj_angular_output(self.projpj, PJ_FWD),
+            PJ_INV: proj_angular_output(self.projpj, PJ_INV),
+            PJ_IDENT: proj_angular_output(self.projpj, PJ_IDENT),
+        }
 
     @staticmethod
-    def _init_crs_to_crs(proj_from, proj_to, skip_equivalent=False):
+    def from_crs(crs_from, crs_to, skip_equivalent=False):
+        crs_from = CRS.from_user_input(crs_from)
+        crs_to = CRS.from_user_input(crs_to)
```

Comment:
Minor remark: I would move this input validation to the python class Transformer.from_crs, since it is python code itself (you know that this method is only called from Tranformer.from_crs/from_proj)

Reply:
Good point - it also allows enforcing the `from_crs` type to be a `_CRS` object. I made the change.